packages/devtools/lib/src/logging/logging.dart

```diff
@@ -705,3 +704,31 @@ class LogDetailsUI extends CoreElement {
     }
   }
 }
+
+class FrameInfo {
+  FrameInfo(this.number, this.elapsedMs, this.startTimeMs);
+
+  static const double kTargetMaxFrameTimeMs = 1000.0 / 60;
+
+  static FrameInfo from(Map<dynamic, dynamic> data) {
```

Comment:
nit: data is really a Map<String, Object>, right?

Reply:
appears so. fixed.